docs/Deploy/Deploy-Fleet-on-Kubernetes.md

```diff
@@ -93,6 +93,8 @@ If you have not used Helm before, you must run the following to initialize your
 helm init
 ```
 
+> Note: The helm init command has been removed in Helm v3. It performed two primary functions. First, it installed Tiller which is no longer needed. Second, it set up directories and repositories where Helm configuration lived. This is now automated in Helm v3; if the directory is not present it will be created.
```

Comment:
Went ahead and put this in a blockquote (just made the change myself to avoid more back-and-forth on formatting)